deploy.sh

```diff
@@ -48,44 +48,44 @@ docker build -t "${REPOSITORY}:${TAG}-amd64" . && \
 docker push "${REPOSITORY}:${TAG}-amd64"
 
 # Create Initial Manifests
-docker manifest create "${REPOSITORY}:${BRANCH}" "${REPOSITORY}:${BRANCH}-amd64"
+docker manifest create "${REPOSITORY}:${TAG}" "${REPOSITORY}:${TAG}-amd64"
 if [[ "$BRANCH" == "latest" ]]; then
-    docker manifest create "${REPOSITORY}:${TAG}" "${REPOSITORY}:${TAG}-amd64"
+    docker manifest create "${REPOSITORY}:${BRANCH}" "${REPOSITORY}:${TAG}-amd64"
 fi
 
 # ARM variants
 for i in $(ls *arm*); do
     ARCH="$(echo ${i} | cut -d. -f2)"
-    docker build -t "${REPOSITORY}:${TAG}-${ARCH}" . && \
+    docker build -f "Dockerfile.${ARCH}" -t "${REPOSITORY}:${TAG}-${ARCH}" . && \
     docker push "${REPOSITORY}:${TAG}-${ARCH}"
     # Add variant to manifest
-    docker manifest create -a "${REPOSITORY}:${BRANCH}" "${REPOSITORY}:${BRANCH}-${ARCH}"
+    docker manifest create -a "${REPOSITORY}:${TAG}" "${REPOSITORY}:${TAG}-${ARCH}"
```

Comment:
This logic repeated is how it will no longer do latest-arch and properly know what to tag